app/controllers/events_controller.rb

```diff
@@ -18,7 +18,7 @@ def show
         event: @event,
         bridge_title: @event.bridge.title,
         bridge_slug: @event.bridge.slug,
-        events: fetch_events.to_json(only: %i[completed completed_at id status_code])
+        events: fetch_events.to_json(only: %i[aborted completed completed_at id status_code])
```

Comment:
I believe this data is for the sidebar? Are we planning on changing the sidebar to say aborted or something? Can't see why we need this.

Reply:
Nah, this is necessary for the events page. 🙂 When an event is aborted the snackbar at the top will say aborted.